torch/amp/autocast_mode.py

```diff
@@ -3,7 +3,7 @@
 import warnings
```

Comment:
I'm a bit confused. So this diff is a BC-breaking change now?

Reply:
I don't think it's BS-breaking. User can still use `torch.autocast` since we import it in [\_\_init\_\_.py](https://github.com/leslie-fang-intel/pytorch/blob/5251aa9cf3e7a70e564737529ded9dbbfdac3421/torch/__init__.py#L718)

Reply:
if it is not BC-breaking, don't update the test paths haha